bolt/internal/internal_test.go

```diff
@@ -179,3 +180,149 @@ func Test_MarshalDashboard(t *testing.T) {
 		t.Fatalf("Dashboard protobuf copy error: diff follows:\n%s", cmp.Diff(dashboard, actual))
 	}
 }
+
+func Test_MarshalDashboard_WithLegacyBounds(t *testing.T) {
+	dashboard := chronograf.Dashboard{
+		ID: 1,
+		Cells: []chronograf.DashboardCell{
+			{
+				ID:   "9b5367de-c552-4322-a9e8-7f384cbd235c",
+				X:    0,
+				Y:    0,
+				W:    4,
+				H:    4,
+				Name: "Super awesome query",
+				Queries: []chronograf.DashboardQuery{
+					{
+						Command: "select * from cpu",
+						Label:   "CPU Utilization",
+						Range: &chronograf.Range{
+							Upper: int64(100),
```

Comment:
Can we remove this and any other reference to queryConfig.range.upper and lower?